skglm/estimators.py

```diff
@@ -1159,6 +1162,169 @@ def fit(self, X, y):
     # TODO add predict_proba for LinearSVC
 
 
+class CoxEstimator(LinearModel):
+    r"""Elastic Cox estimator with Efron and Breslow estimate.
+
+    Refer to :ref:`Mathematics behind Cox datafit <maths_cox_datafit>`
+    for details about the datafit expression. The data convention for the estimator is
+
+    - ``X`` the design matrix with ``n_features`` predictors
+    - ``y`` a two-column vector where the first ``tm`` is of event time occurrences
+      and the second ``s`` is of censoring.
+
+    For L2 Cox, ``l1_ratio=0.``, :ref:`LBFGS <skglm.solvers.LBFGS>`
+    is the used solver, otherwise it is :ref:`ProxNewton <skglm.solvers.ProxNewton>`.
+
+    Parameters
+    ----------
+    alpha : float, optional
+        Penalty strength. It must positive ``alpha > 0``.
+
+    l1_ratio : float, default=0.7
+        The ElasticNet mixing parameter, with ``0 <= l1_ratio <= 1``. For
+        ``l1_ratio = 0`` the penalty is an L2 penalty. ``For l1_ratio = 1`` it
+        is an L1 penalty.  For ``0 < l1_ratio < 1``, the penalty is a
+        combination of L1 and L2.
+
+    method : {'efron', 'breslow'}, default='efron'
```

Comment:
is it the name that lifelines uses?

Reply:
`lifelines` will automatically fall to using `efron` when data contains tied observations (the user cannot contest)

[Here, baseline_estimation_method,](https://lifelines.readthedocs.io/en/latest/fitters/regression/CoxPHFitter.html?highlight=coxph#lifelines.fitters.coxph_fitter.CoxPHFitter)  is the occurrence of the name ``breslow``. On the other hand, ``efron`` is [hard-coded in the codebase](https://github.com/CamDavidsonPilon/lifelines/blob/bcf040caa5a8bc706de316392abf9d4c3626a4d4/lifelines/fitters/coxph_fitter.py#L1380)